zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -240,14 +242,40 @@ object ServerSpec extends HttpRunnableSpec {
           }
       }
   }
+  def requestBodySpec = suite("RequestBodySpec") {
+    testM("POST Request stream") {
+      val app: Http[Any, Throwable, Request, Response] = Http.collect[Request] { case req =>
+        Response(data = HttpData.fromStream(req.bodyAsStream))
```

Comment:
Consider the following example — 
```scala
        Response(data = req.data)
```

I think special case should be handled as an echo operation.

Reply:
Will create an issue for this for Echo when Incoming is received